### PR TITLE
[Post] Add Automating Everything

### PR DESCRIPTION
**Pull request by segiddins**
\c @orta

### CODE REVIEW

**Comment by kylef** [comment]
Looks good to me :+1:

File: _posts/2015-04-06-Automating-Everything.markdown [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+To develop CocoaPods, not only do we have many repositories for different sites and different gems, but we also have a pair of meta-repositories, [Rainforest](https://github.com/CocoaPods/Rainforest) and [Strata](https://github.com/CocoaPods/Strata). They manage the gem and web repos, respectively, and provide a litany of commonly unused functionality, such as updating RuboCop configurations, seeding databases, and bootstrapping all of the repos they manage.
```
Comment (alloy) [inline _posts/2015-04-06-Automating-Everything.markdown]: 'Unused'?

File: _posts/2015-04-06-Automating-Everything.markdown [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+In between CocoaPods 0.36.1 and 0.36.2, I had refactored an internal API to take an array instead of a string, and it just so happening this was an API that interacted with the downloader. Because the CocoaPods `Gemfile` pointed to a more recent commit of `cocoapods-downloader`, a normal run-through of the specs didn't reveal the incompatibility. So I released 0.36.2, and it 'broke' CocoaPods for a few hundred users before 0.36.3 was pushed. All in all, it wasn't too big of a crisis, but it left me with a burning desire to make sure that nothing similar happened again. I was determined to bolster the release task to guard against a repition of this sort of mistake.
```
Comment (alloy) [inline _posts/2015-04-06-Automating-Everything.markdown]: 'Happens'

**Comment by alloy** [comment]
I like!

Can you add a bit about how the PR fixes the issue?

**Comment by orta** [comment]
I'm good after @alloy's fixes 👍

**Comment by orta** [comment]
:shipit:

**Pull request merged**